src/pubd/repository.rs

```diff
@@ -673,8 +732,24 @@ impl RrdpServer {
         let mut session_dir = self.rrdp_base_dir.clone();
         session_dir.push(self.session.to_string());
 
-        for entry in fs::read_dir(&session_dir)? {
-            let entry = entry?;
+        for entry in fs::read_dir(&session_dir).map_err(|e| {
+            KrillIoError::new(
+                format!(
+                    "Could not read RRDP session directory '{}'",
+                    session_dir.to_string_lossy()
+                ),
+                e,
+            )
+        })? {
+            let entry = entry.map_err(|e| {
+                KrillIoError::new(
+                    format!(
+                        "Could not read RRDP session directory '{}'",
```

Comment:
This error message is the same as the one above. Should something be added to make them distinct?

Reply:
ack added 'entry in'